repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEntityStoreV2.java

```diff
@@ -585,6 +591,10 @@ public EntityMutationResponse deleteById(final String guid) throws AtlasBaseExce
 
             AtlasAuthorizationUtils.verifyDeleteEntityAccess(typeRegistry, entityHeader, "delete entity: guid=" + guid);
 
+            if (AtlasConfiguration.NOTIFICATION_ATLAS_DISTRIBUTED_TASKS_TOPIC_NAME.getBoolean()) {
```

Comment:
Suggestion: Instead of repeating this 3 times, I suggest moving this to [this location](https://github.com/atlanhq/atlas-metastore/blob/8582d46173297c6737ac4df79648d83d55802c72/repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEntityStoreV2.java#L2087)